akv-plugin/cmd/get-plugin-metadata.cs

```diff
@@ -0,0 +1,22 @@
+using Notation.Plugin.Proto;
+
+namespace Notation.Plugin.AzureKeyVault.Cmd
+{
+    /// <summary>
+    /// Implementation of get-plugin-metadata command.
+    /// </summary>
+    public class GetPluginMetadata : IPluginCommand
+    {
+        public async Task<object> RunAsync(string _)
+        {
+            return await Task.FromResult<object>(new GetMetadataResponse(
+                name: "akv-plugin",
+                description: "Notation Azure Key Vault plugin",
+                version: "1.0.0",
+                url: "https://github.com/Azure/notation-azure-kv/",
```

Comment:
nit:
```suggestion
                url: "https://github.com/Azure/notation-azure-k",
```

Reply:
Updated to be ` url: "https://github.com/Azure/notation-azure-kv`